### DISCUSSION TITLE
Windows Store?

### DISCUSSION THREAD

**Discussion by DanTheMan827** [cyanfish/naps2#209] [General]
There’s a version on the Windows Store, but I don’t see it referenced anywhere.  Is this an official release?

https://apps.microsoft.com/detail/naps2-document-scanner/XPFPGHZZ8M7MMH

**Comment by cyanfish** [comment]
No, there's some discussion of other ones [here](https://github.com/cyanfish/naps2/issues/51), there's not currently an official Microsoft Store version. I plan on having one but haven't gotten around to finishing the certificate verification process yet.
